requirements.txt

```diff
@@ -1,12 +1,36 @@
-# appJar==0.90.0
-pandas
-flask
-dataset
-rsonlite==0.1.0
-Flask-WTF==0.14.2
+alembic==1.8.1
+autopep8==2.0.1
+banal==1.0.6
+click==8.1.3
+dataset==1.5.2
+decorator==5.1.1
+Flask==2.2.2
+Flask-Migrate==2.5.2
 Flask-SQLAlchemy==2.5.1
+Flask-WTF==0.14.2
+greenlet==2.0.1
+importlib-metadata==5.1.0
+infinity==1.5
+intervals==0.9.2
+itsdangerous==2.1.2
+Jinja2==3.1.2
+Mako==1.2.4
+MarkupSafe==2.1.1
+numpy==1.23.5
+pandas==1.5.2
+pycodestyle==2.10.0
+python-dateutil==2.8.2
+python-dotenv==0.10.3
+pytz==2022.6
+PyYAML==6.0
+rsonlite==0.1.0
+six==1.16.0
+SQLAlchemy==1.4.44
+SQLAlchemy-Utils==0.38.3
+tomli==2.0.1
+validators==0.20.0
+Werkzeug==2.2.2
 WTForms==2.2.1
 WTForms-Alchemy==0.16.9
 WTForms-Components==0.10.4
-Flask-Migrate==2.5.2
-python-dotenv==0.10.3
+zipp==3.11.0
```

Comment:
Separate point: is GH Dependabot running on this repo? Automated dependency update monitoring would be nice